src/components/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -62,7 +62,9 @@ export const ConfirmationModal = ({
       }
       {...props}
     >
-      {children}
+      <div style={{ minWidth: '480px' }}>
```

Comment:
This is problematic, if the screen size is lower than 480px.

Reply:
Thank you for bringing that to my attention. I've addressed the issue by implementing an adjustable minWidth for the content, taking into consideration screen sizes lower than 480px. If you have any further suggestions or if additional changes are needed, please feel free to let me know.

https://github.com/canonical/react-components/assets/90577359/56f8c342-04b2-41c2-a424-3c100739f361